content/config.yml

```diff
@@ -1,223 +1,859 @@
+    priority: 'other'
+    status: 'updated'
+    tagline: 'Curration Tools for Teachers'
+    description: |
+      We worked with Junyo to develop
+      an early version of their Streams Resource Library —
+      a community-currated directory of teaching materials.
+      Teachers can collect and access
+      all of their open and licensed digital resources in one library.
+      Search by curriculum, student need,
+      use case, content efficacy, users’ rankings,
+      standards, and other criteria,
+      to create custom learning streams.
+      OddBird built the initial curation and search tools
+      used to populate, annotate, and sort resources.
+    client:
+      name: 'Junyo'
+      url: 'http://junyo.com/'
+
+  - name: 'Portfoliyo'
+    source: 'https://github.com/oddbird/portfoliyo'
+    years: '2012-2013'
+    priority: 'other'
+    status: 'removed'
+    tagline: 'Real-Time Communication for Parents, Teachers, and Students'
+    client:
+      name: 'The K Network'
+
+  - name: 'MozTrap'
+    source: 'https://github.com/mozilla/moztrap'
+    url: 'http://moztrap.mozilla.org'
+    years: '2011-2012'
+    feature: true
+    priority: 'live'
+    tagline: 'Ditributed Quality Assurance Tools'
+    description: |
+      Mozilla's next-generation test case management system,
+      MozTrap manages extensive test suites
+      for all of Mozilla's software products
+      (including Firefox, Thunderbird, Firefox OS, and others),
+      and allows community testers from around the world
+      to run tests on their system and report results back to Mozilla,
+      which MozTrap aggregates into reports for product managers.
+      MozTrap also includes a RESTful JSON API
+      to allow automated tools to run tests and report results.
+    client:
+      name: 'Mozilla'
+      url: 'https://www.mozilla.org/'
+
+  - name: 'Master Lookup Table'
+    source: 'https://github.com/oddbird/mlt'
+    years: '2011'
+    feature: true
+    priority: 'live'
+    tagline: 'Parcel Mapping for Urban Development'
+    description: |
+      With custom interactive in-browser maps,
+      MLT streamlines the Providence Plan staff's task
+      of mapping street addresses to deeded land parcels.
+      In addition to allowing batch imports
+      of large numbers of addresses from CSV files
+      recording user-selected associations between addresses and parcels,
+      MLT records an audit trail of all data changes
+      and allows one-click rollback of any change.
+      We worked closely with ProvPlan staff
+      to ensure the tool was easy to understand,
+      efficient, beautiful, and a pleasure to work with.
+      MLT also includes a RESTful JSON API
+      for querying address mapping data.
+    client:
+      name: 'The Providence Plan'
+      url: 'http://provplan.org/'
+
+  - name: 'Craftisan'
+    years: '2010-2011'
+    priority: 'other'
+    status: 'updated'
+    client:
+      name: 'Craftisan'
+      url: 'http://craftisan.in/'
```

Comment:
A different company doing exactly the exact same thing... Odd... Guess I didn't look close enough.